esmvaltool/references/esdc.bibtex

```diff
@@ -0,0 +1,11 @@
+@Article{esdc,
```

Comment:
Is there anything I could do? It's not clear to me how to change that...

Reply:
Yes, I think it would be better to only use lowercase and also not having the "doi" on the last line. I think the Tool misses the doi because the last line does not end with a comma. You could follow the format of [this example](https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/references/merra2.bibtex).